pkg/actions/service.go

```diff
@@ -310,10 +306,10 @@ func buildRunManifestFromTasks(record graveler.HookRecord, tasks [][]*Task) RunM
 				ActionName: task.Action.Name,
 				StartTime:  task.StartTime,
 				EndTime:    task.EndTime,
-				Passed:     task.Err == nil,
+				Passed:     task.Err == nil && !task.StartTime.IsZero(), // mark skipped tasks as failed
 			})
-			// keep min run start time
-			if manifest.Run.StartTime.IsZero() || task.StartTime.Before(manifest.Run.StartTime) {
+			// keep min run start time using non-skipped tasks
+			if manifest.Run.StartTime.IsZero() || (!task.StartTime.IsZero() && task.StartTime.Before(manifest.Run.StartTime)) {
```

Comment:
1. Using a variable called 'taskStarted' set by '!manifest.Run.StartTime.IsZero()' it can simplify the code and reduce the number of checks
2. Previous commit also skipped the run end time

Reply:
1. Do you mean `taskStarted` can be set by `!task.StartTime.IsZero()`
2. I am not sure what you meant by previous commit?

Reply:
1. Yes
2. Previous commit in this pr, you checked if the task didn't start and skipped the set of start and end time

Reply:
For 2. I realized that I don't need to do for endtime because when the condition is to check .After so it won't affect it